src/js/profile/mobile/widget/Calendar.js

```diff
@@ -172,40 +218,90 @@
 				var calendarElement = this.element.querySelector(".calendarView");
 
 				if (event.srcElement.className === "calendarArrowRight") {
-					todayMonth = todayMonth + 1;
-					if (todayMonth === 13) {
-						todayMonth = 1;
-						todayYear = todayYear + 1;
-					}
-					prototype._deleteCalendar(calendarElement);
-					today = new Date(todayYear, todayMonth - 1);
-					prototype._buildCalendar(calendarElement);
+					prototype._moveMonth("right", calendarElement);
```

Comment:
use `this` or `self` instead of `prototype`

Reply:
done.